init.lua

```diff
@@ -67,21 +67,6 @@ end)
 -- Case-insensitivity --
 ------------------------
 
-minetest.register_on_prejoinplayer(function(name, ip)
-	if player_exempted(name) then return end
-
-	-- Check for used names
-	local lname = name:lower()
-	for iname, data in pairs(minetest.auth_table) do
-		if iname:lower() == lname and iname ~= name then
-			return "Sorry, someone else is already using this"
-				.." name.  Please pick another name."
-				.."  Annother possibility is that you used the"
-				.." wrong case for your name."
-		end
-	end
-end)
-
```

Comment:
Could you add this back in a fixed form?

Reply:
Yes of course I can but first can you tell me why you want to duplicate a function in the auth handler? This was the reason I removed it...
https://github.com/minetest/minetest/blob/master/builtin/game/auth.lua#L130

Reply:
It's stated in the comment for the pr, it's courteous to explain why you know it should be added back in to the pr, because clearly I don't :smiley:

Reply:
I can wait another 4 months for a reply if you like...